frontend/src/Components/Form/MultiSelectInputOption.js

```diff
@@ -0,0 +1,76 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import EnhancedSelectInputOption from './EnhancedSelectInputOption';
+import CheckInput from './CheckInput';
+import styles from './MultiSelectInputOption.css';
+
+class MultiSelectInputOption extends Component {
+
+  //
+  // Listeners
+
+  onPress = () => {
```

Comment:
Nit: not needed.

Reply:
CheckInput requires an event handler, and normally it's absolutely required to be an effective control. But we kinda abuse it to show a check symbol, so I didn't want to remove the isRequired from props there.